test/pdfs/MBTA-pretax-form-July2012.pdf.link

```diff
@@ -0,0 +1 @@
+https://web.archive.org/web/20121105185256if_/http://www.northeastern.edu/hrm/pdfs/resources/benefits/MBTA-pretax-form-July2012.pdf
```

Comment:
The `if_` part should be removed here, since that will be added *automatically* by https://github.com/mozilla/pdf.js/blob/master/test/downloadutils.js

Furthermore, please add a blank line at the end of the file (to avoid the big red marker here in the Github diff).

Reply:
Fixed.